js/bybit.js

```diff
@@ -592,6 +592,7 @@ module.exports = class bybit extends Exchange {
                     // '30084': BadRequest, // Isolated not modified, see handleErrors below
                     '33004': AuthenticationError, // apikey already expired
                     '34026': ExchangeError, // the limit is no change
+                    '34036': BadRequest, // {"ret_code":34036,"ret_msg":"leverage not modified","ext_code":"","ext_info":"","result":null,"time_now":"1652376449.258918","rate_limit_status":74,"rate_limit_reset_ms":1652376449255,"rate_limit":75}
```

Comment:
We might need to introduce a new error here, similar to MarginModeAlreadySet, something like LeverageAlreadySet:

- https://github.com/ccxt/ccxt/blob/master/js/base/errorHierarchy.js#L15
- https://github.com/ccxt/ccxt/blob/master/js/binance.js#L5162